docs/production-deployment/cloud/audit-logging.mdx

```diff
@@ -23,7 +23,7 @@ tags:
   - troubleshooting
 ---
 
-Audit Logging is a feature of [Temporal Cloud](/cloud/overview) that provides forensic access information at the account level, the user level, and the [Namespace](/namespaces) level.
+Audit Logging is a feature of [Temporal Cloud](/cloud/overview) that provides forensic access information at the account level, the user level, [Namespace](/namespaces) level, and the Nexus Endpoint level.
```

Comment:
```suggestion
[Temporal Cloud](/cloud/overview) Audit Logging provides forensic access information at the account level, the user level, [Namespace](/namespaces) level, and the Nexus Endpoint level.
```

If these are new terms, can we get them into Vale?

Reply:
added jira ticket